app/embeds/GoogleForms.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from "react";
+import Image from "components/Image";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+  "^https?://docs.google.com/forms/d/e?/(.*)/(viewform|edit)?$"
+);
+const URL_REGEX_MINIFIED = new RegExp("^https?://forms.gle/(.*)$");
+
+type Props = {|
+  attrs: {|
+    href: string,
+    matches: string[],
+  |},
+|};
+
+export default class GoogleForms extends React.Component<Props> {
+  static ENABLED = [URL_REGEX] || [URL_REGEX_MINIFIED];
```

Comment:
Similar comment to the other PR here, this syntax doesn't make sense – the second case will never be hit.